tests/common.go

```diff
@@ -255,6 +255,8 @@ const (
 	defaultCmdTimeout         = 20 * time.Second
 	defaultCmdRetryInterval   = 5 * time.Second
 	defaultDriverStartTimeout = 10 * time.Minute
+	defaultCommandRetry       = 5 * time.Second
```

Comment:
There's nearly identical variables defined above this: any reason we can't use those?

Reply:
Done, removed and reused the one's which are available.